packages/cli/src/lib/build-envs/wasm/rust/Dockerfile.mustache

```diff
@@ -0,0 +1,79 @@
+FROM rust:{{rust_version}} as base
+
+# Install nightly rust
+RUN rustup toolchain install nightly
```

Comment:
Are you referring to `{{rust_version}}` in the Mustache file? That's a placeholder for the specific version being used.

Reply:
I was referring to "RUN rustup toolchain install nightly" , though the {{rust_version}} has the same side effect since supposedly the build of the version can change. There was talk about it here https://youtu.be/uWBnO11Y7bc?t=1123 
Suggest fix was using hash of the version instead

Reply:
Ah okay. `rustup toolchain install nightly` defaults to the latest nightly release, though.